jupyterhub/files/hub/jupyterhub_config.py

```diff
@@ -21,7 +29,7 @@
 c.JupyterHub.spawner_class = 'kubespawner.KubeSpawner'
 
 # Connect to a proxy running in a different pod
-c.ConfigurableHTTPProxy.api_url = 'http://{}:{}'.format(os.environ['PROXY_API_SERVICE_HOST'], int(os.environ['PROXY_API_SERVICE_PORT']))
+c.ConfigurableHTTPProxy.api_url = 'http://proxy-api'
```

Comment:
It seems like the proxy-api service is exposed at 8001 rather than 80

Reply:
fixed